.drone.yml

```diff
@@ -9,6 +9,27 @@ clone:
     tags: true
 
 pipeline:
+  download_translations:
+    image: jonasfranz/crowdin
+    pull: true
+    secrets: [ crowdin_key ]
+    project_identifier: gitea
+    ignore_branch: true
+    download: true
+    export_dir: options/locale/
+    when:
+      event: [ push ]
+      branch: [ master ]
+  git_push:
+    image: appleboy/drone-git-push
+    pull: true
+    secrets: [ git_push_ssh_key ]
```

Comment:
And this?

Reply:
Yes you should add the gut_push_ssh_key like described in the plug-in documentation.

Reply:
I have added the `git_push_ssh_key` in drone secret.